cmd/start.go

```diff
@@ -63,17 +66,12 @@ var startCmd = &cobra.Command{
 func init() {
 	// Register the start command
 	RootCmd.AddCommand(startCmd)
-	startCmd.Flags().StringVarP(&privateKeyFileName, privateKeyFileNameFlag, "f", "", "filename of the private key file")
-	_ = startCmd.MarkFlagRequired(privateKeyFileNameFlag)
 
+	startCmd.Flags().StringVarP(&privateKeyFileName, privateKeyFileNameFlag, "f", "", "filename of the private key file")
 	startCmd.Flags().StringVarP(&privateKeyPassword, privateKeyPasswordFlag, "P", "", "password of the private key")
```

Comment:
Why is the password the only one with a capital letter shortened? (I know it's unchanged code but just noticed 🙂  )

Reply:
small `p` is for port

Reply:
👍